server/pairing/payload_receiver.go

```diff
@@ -40,7 +40,7 @@ type BasePayloadReceiver struct {
 	receiveCallback func()
 }
 
-func NewBaseBasePayloadReceiver(e *PayloadEncryptor, um ProtobufUnmarshaller, s PayloadStorer, callback func()) *BasePayloadReceiver {
+func NewBasePayloadReceiver(e *PayloadEncryptor, um ProtobufUnmarshaller, s PayloadStorer, callback func()) *BasePayloadReceiver {
```

Comment:
🤣

[![Buttery Biscuit Base](https://github.com/status-im/status-go/assets/5702426/65f3c7e6-1c99-45cc-860b-0eac226a985d)](https://youtu.be/IfeyUGZt8nk)
![image](https://github.com/status-im/status-go/assets/5702426/ddb26ac5-dadb-45ed-8e88-848242bd38c7)